### ISSUE TITLE
Can this speech SAPI5 voices on Linux? How can I use LTTS7 (Loquendo) voices with this?

### ISSUE THREAD

**Issue by z3nth10n** [espeak-ng/espeak-ng#836]
Hi,

I'd like to run this utility in order to speech texts using Loquendo LTTS7 voices on Linux. I have a huge mess trying to use Balabolka on Linux with Wine, but this is awful...

For this reason, I'd like t know if there is a possible solution for this on Linux with this utility or not.

Thanks!
PD: Is there available any Discord / gitter chat?

Best regards.

**Comment by 0x5066** [comment]
eSpeak-NG doesnt allow you to use SAPI5 voices on linux directly, if that's what you're asking

**Comment by z3nth10n** [comment]
Yep this is what I'm trying. I think I'll try: https://github.com/mjakal/sapi5_on_linux

Thanks anyway!

**Comment by z3nth10n** [comment]
I would like to ask something more. I have discovered that there are binaries for Linux in Loquendo. Could be these binaries used in Linux.

I would like to know if there is an option to use this on Linux which any method.

I share the question I posted on Stackoverflow to let you know which methods I tried: https://stackoverflow.com/questions/64934405/trying-to-find-the-original-loquendo-products-for-linux-not-windows

Thanks!
